### ISSUE TITLE
Problem importing Iron Man 20.INH

### ISSUE THREAD

**Issue by weidercs** [evilhero/mylar#1092]
Trying to import Iron Man (2014) 20.INH, it looks like it finds it correctly, then during the matching process something makes it 'not match' and thus doesn't download/import.

```
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : checking search result: Iron Man 020.INH (2014) (Digital) (Zone-Empire)
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : size given as: 40.9 MB
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : Thu, 23 Jan 2014 07:57:08 +1000 is after store date of 2014-01-22
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : Entry: Iron Man 020.INH (2014) (Digital) (Zone-Empire)
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : Cleantitle: iron man 020 inh (2014) (digital) (zone-empire)
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : Cleantitle: iron man 020 inh (2014) (digital) (zone-empire)
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : 0. Bracket Word: iron man 020 inh 
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : Comic: iron man 020 inh 
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : UseFuzzy is  : None
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : ComVersChk : 6
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : 1. Bracket Word: 2014
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : year detected: 2014
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : year looking for: 2014
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : 2014 - right years match baby!
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : 3. Bracket Word: digital
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : digital edition detected
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : 5. Bracket Word: zone-empire
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : Scanner detected: zone-empire
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : original nzb comic and issue: iron man 020 inh 
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : adjusted nzb comic and issue: iron man 020 inh 
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : alphas: inh
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : a_issno: i
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : adjusting from: iron man 020 inh  to: inh
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : ['iron', 'man', '020', 'inh'] nzb series word count: 3
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : ['iron', 'man'] watchlist word count: 2
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : splitst : 3
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : len-watchcomic : 2
22-Jul-2015 22:18:31 - DEBUG   :: Thread-18 : incorrect comic lengths after removal...not a match.
```

**Comment by evilhero** [comment]
Yeah the inh issues were and still are problematic mainly because CV doesn't actually list some as inh and some posters posted them either with or without. Just really confusing.

I'll have to look into this abit later and see, it was working at one point but so much has changed in the modules lately it probably got broken.

Just to clarify this was off a search correct?

**Comment by weidercs** [comment]
Yeah, I posted it over to the forum before, but thought it was easier tracked here...  Simple adding off the Iron Man (2013) series, doesn't pick up 20.INH, it finds it fine from the indexer but fails on the match.

**Comment by weidercs** [comment]
FIXED!

**Comment by weidercs** [comment]
Closing as fixed in 60140e8c900ae6c2acc3c9bf319506b6514fbed4
